lib/plugins/wallet-scoring/scorechain/scorechain.js

```diff
@@ -0,0 +1,77 @@
+const axios = require('axios')
+const _ = require('lodash/fp')
+
+
+const NAME = 'Scorechain'
+const SUPPORTED_COINS = {
+  BTC: 'BITCOIN',
+  ETH: 'ETHEREUM',
+  USDT: 'ETHEREUM',
+  BCH: 'BITCOINCASH',
+  LTC: 'LITECOIN',
+  DASH: 'DASH',
+  TRX: 'TRON',
+  USDT_TRON: 'TRON'
+}
+
+const TYPE = {
+  TRANSACTION: 'TRANSACTION',
+  ADDRESS: 'ADDRESS'
+}
+
+function rate (account, objectType, cryptoCode, objectId) {
+  if (!isWalletScoringEnabled(account, cryptoCode)) return Promise.resolve(null)
```

Comment:
`isWalletScoreingEnabled()` returns a promise :/ I could have avoid this by being more explicit...

Reply:
nah, it's my bad. 

I don't think something called `isWalletScoringEnabled` should be a promise, but it's a bigger change for that one.